accessing-data/record-webhooks.mdx

```diff
@@ -129,8 +129,94 @@ Body:
       record_slug: 'ffd928c3-3459-43fb-a2e4-3f1b3ad5fc1f',
       validation_type: 'error',
       column_name: 'email',
-      issue_description: 'Email already exists and must be unqiue.'
+      issue_description: 'Email already exists and must be unique.'
     }]
   }
 }
-```
\ No newline at end of file
+```
+
+## Record Webhook Implementation Example
+
+In this example using Javascript and Express, we create an endpoint in your backend that initially receives the token `http_x_fuse_signature_256` containing the signature generated by Fuse and submitted to you. \
+You can use this signature to confirm the authenticity of the data, then, compare the signature with a generated one using the payload and your secret key.
+
+After that, we start validating the data, creating an empty array to store the possible issues, retrieving the `import_slug` for further usage and looping through the received data.
+
+In this step, you can do any validation you may need with the data, such as checking property existence in your backend, API calls, comparing values, etc. \
+For this example, we have two mock functions to simulate the data being validated: `validateIdExistsInTheBackend` and `validateEmailUniquenessInServer`.
+
+If you need to create some issue to be shown in the UI after the webhook is processed, you can use this structure below:
+
+```javascript
+{
+  import_slug: 'f50df76a-e5b9-4784-80ba-6ef27948e2ba',
+  record_slug: 'cb28e820-8af5-4f3a-b01b-87f2eb3f5014',
+  validation_type: 'error',
+  column_name: 'id',
+  issue_description: 'An error occurred for the ID column.',
+}
+```
+
+- `import_slug` can be accessed through `req.body.import_slug`.
+- `record_slug` can be accessed from the current position of the `req.body.records` array, doing `req.body.records[index].slug`.
+- `validation_type` can be "error" or "warning".
+- `column_name` should be the exact `internal_key` you defined in your Template columns implementation that you want to attach the validation.
+- `issue_description` is the message you want to show in the UI after the webhooks are processed.
+
+Finally, we send a response from our webhook to Fuse in the request done, sending the batch status as completed and sending the `validation_issues`.
+
+```javascript
+router.post('/record-webhook', async (req, res) => {
```

Comment:
Use the Code Groups UI to show code samples if different programming languages.
https://mintlify.com/docs/content/components/code-groups

If there are other areas of the docs that need this update please do it.